### PR TITLE
Remove .osm workflow from workers

### PR DESCRIPTION
**Pull request by TShapinsky**
#201 
There is still some OSM language existing in alfalfa. Some is dealing with the seed models for OSWs and some is on the front end. I haven't touched the front end due to being unclear as to what is actually referring OSMs and what is an OSW workflow condition that is called "osm".

### CODE REVIEW

File: alfalfa_worker/worker_fmu/add_site.py [hunk 1]
```diff
@@ -141,38 +139,6 @@ def insert_os_tags(self, points_json_path, mapping_json_path):
         # add points to database
         self.ac.add_site_to_mongo(points_json, self.upload_id)
 
-    def add_osm(self):
```
Comment (nllong) [inline alfalfa_worker/worker_fmu/add_site.py:144]: 🧹!

**Review by nllong — COMMENTED** [review]
Looking good. Note that #223 will remove some of the osmName items. I haven't merged yet because we'll need to release an updated version of alfalfa-client.

**Comment by TShapinsky** [comment]
@nllong just merged in the osm rename, everything appears to be working on my end

**Comment by nllong** [comment]
> @nllong just merged in the osm rename, everything appears to be working on my end

noice! this is going to be an amazing PR with nice cleanup!

**Comment by nllong** [comment]
@TShapinsky -- is this still a WIP, or can we merge?

**Comment by TShapinsky** [comment]
> @TShapinsky -- is this still a WIP, or can we merge?

I think this is good if we're ready to merge the client change as well

**Comment by nllong** [comment]
sweet, let me do a review of it by tomorrow's meeting. 

I'm happy merging the client down, but will want to wait to release. Do you think it is an issue having people (via poetry) use the Github version of the client?

**Comment by nllong** [comment]
Also, I think [the workflow.tar.gz](https://github.com/NREL/alfalfa/blob/201-remove-osm-workflow/alfalfa_worker/workflow.tar.gz) can be removed since it was used in the OSM workflow? True?

That also means `def extract_workflow_tar(self):` can disappear.

**Comment by TShapinsky** [comment]
> Also, I think [the workflow.tar.gz](https://github.com/NREL/alfalfa/blob/201-remove-osm-workflow/alfalfa_worker/workflow.tar.gz) can be removed since it was used in the OSM workflow? True?
> 
> That also means `def extract_workflow_tar(self):` can disappear.

I can make those changes and see if the tests still pass

**Comment by TShapinsky** [comment]
@nllong we also still have `sim_osm.py` and `step_osm.py` and `osm_model_advancer` should I change all of those to osw?

**Comment by nllong** [comment]
> @nllong we also still have `sim_osm.py` and `step_osm.py` and `osm_model_advancer` should I change all of those to osw?

I would leave those as is because I'm going to rename the files to just "simulate.py" and "step.py" or something like that. That way the worker "API" just knows to call simulate in the shared mounted directory for FMU or OSWs.

**Comment by TShapinsky** [comment]
@nllong added the changes, everything appears to still work

**Comment by nllong** [comment]
> 



> @nllong added the changes, everything appears to still work

sweet, can you fix the pre-commit checks... probably formatting.

**Comment by TShapinsky** [comment]
> > 
> 
> > @nllong added the changes, everything appears to still work
> 
> sweet, can you fix the pre-commit checks? probably formatting.

Looks like I didn't have pre-commit set up on my machine. Should pass now

**Review by nllong — APPROVED** [review]
looks great! Thanks!

**Pull request merged**